python/tank/api.py

```diff
@@ -274,9 +274,15 @@ def template_from_path(self, path):
             return matched[0]
         else:
             # ambiguity!
+            # We're erroring out anyway, take the time to create helpful debug info!
+            fields = []
+            for template in matched:
+                fields.append(template.validate_and_get_fields(path))
```

Comment:
You should be ok to just use template.get_fields() here as the template has already been validated